wallet/lib/pages/purchase_item/widgets/pay_now_dialog.dart

```diff
@@ -129,78 +118,56 @@ class _PayNowWidgetState extends State<PayNowWidget> {
               mainAxisAlignment: MainAxisAlignment.center,
               crossAxisAlignment: CrossAxisAlignment.stretch,
               children: [
-                ListTile(
-                  leading: SizedBox(
-                    height: 50.h,
-                    width: 50.h,
-                    child: CachedNetworkImage(
-                      imageUrl: widget.nft.url,
-                      width: 1.sw,
-                      errorWidget: (a, b, c) => Center(
-                          child: Text(
-                        "unable_to_fetch_nft_item".tr(),
-                        style: Theme.of(context).textTheme.bodyText1,
-                      )),
-                      height: 1.sh,
-                      fit: BoxFit.fitHeight,
-                      placeholder: (context, _) => Shimmer(
-                        color: PylonsAppTheme.cardBackground,
-                        child: Container(),
-                      ),
-                    ),
-                  ),
-                  title: Text(
-                    widget.nft.name,
-                    style: _titleTextStyle,
-                    maxLines: 1,
-                  ),
-                  subtitle: Column(
-                    crossAxisAlignment: CrossAxisAlignment.stretch,
-                    mainAxisSize: MainAxisSize.min,
-                    children: [
-                      Text(
-                        "${"created_by".tr()} ${widget.nft.creator}",
-                        style: _subtitleTextStyle,
-                      ),
-                      if (widget.nft.owner.isNotEmpty)
-                        Text(
-                          "${"sold_by".tr()} ${widget.nft.owner}",
-                          style: _subtitleTextStyle,
-                        ),
-                    ],
-                  ),
+                SizedBox(
+                  height: 60.h,
                 ),
+                Padding(
+                    padding: EdgeInsets.only(left: 20.w),
+                    child: Text("checkout".tr(), style: _titleTextStyle)),
                 SizedBox(
                   height: 30.h,
                 ),
                 buildRow(
-                  subtitle: widget.nft.ibcCoins.getAbbrev(),
-                  title: "currency".tr(),
+                  subtitle: widget.nft.name,
+                  title: "NFT",
                 ),
                 SizedBox(
                   height: 3.h,
                 ),
                 buildRow(
-                  subtitle: widget.nft.ibcCoins.getCoinWithDenominationAndSymbol(price.toString(), showDecimal: true),
-                  title: "price".tr(),
+                  subtitle: "#${widget.nft.amountMinted} of ${widget.nft.quantity}",
```

Comment:
widget.nft.amountMinted + 1 should be shown. 

Instead of showing 0 of 50 we should show 1 of 50